### DISCUSSION TITLE
When deploying to Vercel, does the runtime environment automatically transition to the Edge Runtime?

### DISCUSSION THREAD

**Discussion by Eunkyung-Son** [vercel/vercel#11082] [Help] [answered]
### Question

Hello, I am using the Vercel Pro plan.
I checked the official documentation and found that the default runtime environment is specified as Node.js.
I'm curious whether the runtime environment automatically transitions to the Edge Runtime when deploying with Vercel.
Do I need to specify export const runtime = 'edge' for each function to use the Edge Runtime?

**Comment by leerob** [comment, marked as answer]
> Do I need to specify export const runtime = 'edge' for each function to use the Edge Runtime?

Yes, for example:

```js
export const config = {
  runtime: 'edge',
};
 
export function GET(request: Request) {
  return new Response(`Hello from ${process.env.VERCEL_REGION}`);
}
```

https://vercel.com/docs/functions/edge-functions/quickstart
